### ISSUE TITLE
Ladybird endlessly loops on `static Gfx::Font& Gfx::FontDatabase::default_font(): Assertion `s_default_font' failed.` unless called via `ninja`

### ISSUE THREAD

**Issue by ctrlcctrlv** [SerenityOS/serenity#16859]
Works fine via `ninja` tho.

Kewl browser.

![image](https://user-images.githubusercontent.com/838783/211042651-5f3eb3ef-86f1-4c7d-ae9f-dc91ec6ca397.png)

**Comment by BertalanD** [comment]
What do you mean "unless called via `ninja`"? Just opening the `ladybird` executable from the build directory?

Ladybird currently uses [three methods](https://github.com/SerenityOS/serenity/blob/7c2e134abdd8532f7d496d999aa455e5a0fc3b32/Ladybird/Utilities.cpp#L33-L45) to locate some of the resources that it needs:
- the location of the SerenityOS source tree can be specified via the `SERENITY_SORUCE_DIR` environment variable. This is set up automatically when you launch Ladybird via the `run` Ninja target.
- If the variable is not set, resources can be stored in `$HOME/.lagom` (not sure when this is used)
- Lastly, if you are doing a system-wide install (via the `install` Ninja target), it will look in `../share` relative to the Ladybird directory. This is done so that e.g. you can install Ladybird as `/usr/local/bin/ladybird`, and resources will be automagically picked up from `/usr/local/share`.

If you are manually running the `ladybird` executable from the build directory, none of these cases apply.

**Comment by ctrlcctrlv** [comment]
@BertalanD Yes that's what I meant. After the build recommended in the instructions, running `cd Build/ladybird && ./ladybird`.
